docs/user_guide/backup/index.md

```diff
@@ -6,224 +6,75 @@ sidebar: sidebars/documentation.html
 
 # Backup
 
-## Prepare Backup
-
-In the following sections we will be using `/backup` directory as our targed directory that will hold backup archives. Let's prepare it now.
+In the following sections we will be using `/tmp/backup` directory as our targed directory that will hold backup archives.
 
 All the following commands are executed under `root` system account.
 
-```
-export BDIR=/backup
-umask 0027
-mkdir -p $BDIR
-chgrp postgres $BDIR
-chmod 770 $BDIR
-cd $BDIR
-```
-
-## Backup system files
+## Option One: Offline repositories backup
 
-It is necessary to backup configuration files and important data files. Being in the `$BDIR` execute:
+By default, the whole Katello instance will be turned off completely for the entire backup.
 
 ```
-tar --selinux -czvf config_files.tar.gz \
-/etc/foreman-installer \
-/etc/foreman \
-/etc/elasticsearch \
-/etc/candlepin \
-/etc/pulp \
-/etc/pki/katello \
-/etc/pki/pulp \
-/etc/gofer \
-/etc/qpid \
-/etc/qpidc.conf \
-/etc/qpidd.conf \
-/etc/sysconfig/katello \
-/etc/sysconfig/foreman \
-/etc/sysconfig/elasticsearch \
-/root/ssl-build \
-/var/www/html/pub \
-/usr/share/katello/candlepin-cert.crt
-
-tar --selinux -czvf elastic_data.tar.gz /var/lib/elasticsearch
+katello-backup /tmp/backup
```

Comment:
not sure if this is consistent with the rest of the docs (did not bother to look), but I like to prepend shell commands entered as root with a # and ones entered as user with $, so one can distinguish input and output, esp usefull in the later sections where you use ls